prompttools/experiment/experiments/epsilla_experiment.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) Hegel AI, Inc.
+# All rights reserved.
+#
+# This source code's license can be found in the
+
+# LICENSE file in the root directory of this source tree.
+
+import pandas as pd
+from typing import Callable, Optional
+
+try:
+    from pyepsilla import vectordb
+except ImportError:
+    vectordb = None
+import logging
+
+from time import perf_counter
+from .experiment import Experiment
+from ._utils import _get_dynamic_columns
+
+VALID_TASKS = [""]
+
+
+
+class EpsillaExperiment(Experiment):
+    r"""
+    Perform an experiment with ``Epsilla`` to test different embedding functions or retrieval arguments.
+    You can query from an existing collection, or create a new one (and insert documents into it) during
+    the experiment. If you choose to create a new collection, it will be automatically cleaned up
+    as the experiment ends.
+
+    Args:
+        epsilla_client (vectordb.Client): Epsilla client to interact with your database
+        collection_name (str): the collection that you will get or create
+        use_existing_collection (bool): determines whether to create a new collection or use
+            an existing one
+        query_collection_params (dict[str, list]): parameters used to query the collection
+            Each value is expected to be a list to create all possible combinations
+        embedding_fns (list[Callable]): embedding functions to test in the experiment
+            by default only uses the default one in Epsilla
+        embedding_fn_names (list[str]): names of the embedding functions
+        add_to_collection_params (Optional[dict]): documents or embeddings that will be added to
+            the newly created collection
+    """
+
+    PARAMETER_NAMES = ["epsilla_client"]
+
+    def __init__(
+        self,
+        epsilla_client: "vectordb.Client",
+        collection_name: str,
+        use_existing_collection: bool,
+        query_collection_params: dict,
+        embedding_fns: list[Callable] = [
+            epsilla.utils.embedding_functions.DefaultEmbeddingFunction if epsilla else None
+        ],
+        embedding_fn_names: list[str] = ["default"],
+        add_to_collection_params: Optional[dict] = None,
+    ):
+        if epsilla is None:
```

Comment:
```suggestion
        if vectordb is None:
```

I think it needs to be `vectordb` since that is the name that you import with